biochemist-python/chapters/introduction.ipynb

```diff
@@ -85,23 +85,12 @@
   },
```

Comment:
<p>I think we had a comment on the evaluation on defining variables more than once. We have <code>s_concs</code>  defined already, so I don't see the need to type it again here. In practice should this be a cell that they keep editing as the session continues?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I changed the text in the last sentence in the markdown cell above this <code>for loop</code> code cell to read:</p><p> </p><p>Let's use a loop to calculate initial velocities for all the substrate concentration in our s_concs list that was created above.</p><p> </p><p>Then I removed </p><p> </p><p>s_concs = [1.0, 3.0, 10.0, 30.0, 100.0]</p><p> </p><p>from the code cell. </p><p> </p><p>In response to your last question, I think it will work best to just refer to the code cells above for the s_concs list and the prior for loop.</p>